data/mods/Xedra_Evolved/npc/xe_liam_chat.json

```
@@ -0,0 +1,66 @@
+[
+  {
+    "type": "talk_topic",
+    "id": "TALK_FRIEND_Liam_CHAT",
+    "dynamic_line": [
+      "Before things ended did you see any of the interviews with some of the remaining Wampanoag?  They kept saying things about other races coming back to this planet.  That's crazy, right?",
+      "A few nights after I got here, I saw lights in the woods off in the distance.  I don't know why, but I ran back into the house turned off all the lights and closed the blinds.  It was just so eerie!"
+    ],
+    "speaker_effect": { "effect": [ "morale_chat_activity", { "npc_add_effect": "asked_to_socialize", "duration": 7000 } ] },
+    "responses": [
+      {
+        "text": "We're always gonna be friends, right?  Even with the apocalypse… changing us?",
+        "topic": "TALK_FRIEND_Liam_CHAT_Friends4ever"
+      },
+      { "text": "What do you miss most about the internet?", "topic": "TALK_FRIEND_Liam_CHAT_Internet" },
+      {
+        "text": "Now that we've got no responsibilities, is there anything you'd like to see or do?",
+        "topic": "TALK_FRIEND_Liam_CHAT_Goals"
+      },
+      {
+        "text": "What do you think the rest of the world is like these days?",
+        "topic": "TALK_FRIEND_Liam_CHAT_World"
+      },
+      { "text": "Do you miss any food from the old days?", "topic": "TALK_FRIEND_Liam_CHAT_Food" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_FRIEND_Liam_CHAT_World",
+    "dynamic_line": [
+      "Things just kept getting weirder and weirder, with strange trees and species showing up in the deep woods no one had ever seen before.  The Border Patrol made statements about 'species previously thought extinct' why the heck was the Border Patrol of all agencies closing off access to forests with endangered species?",
```

```suggestion
      "There could be anything out there.  Remember towards the end before the apocalypse?  I remember tons of weird stuff I heard on the radio.  Like, I heard out past Bunyan there were strange trees and species showing up in the deep woods, stuff no one had ever seen before.  The Border Patrol was going in about 'species previously thought extinct' or something like that.  Didn't make sense to me.  Why the heck was the *Border Patrol* talking about *closing off forests*?  Weird stuff.  I guess that means I don't have a clue what's out there now.",
```

This one seems weird to me as an answer to the question "what do you think the rest of the world is like these days". Suggestion to make it flow better.